### PR TITLE
Add riemann.elasticsearch

### PR DESCRIPTION
**Pull request by eguven**

### CODE REVIEW

File: src/riemann/elasticsearch.clj [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            [cheshire.core :as json]
+            [clj-time.coerce :as time-coerce]
+            [clj-time.format :as time-format]))
+
+(defn- datetime-from-event
+  "Returns the datetime from event correcting (secs -> millisecs) before conversion."
+  [event]
+  (time-coerce/from-long (* 1000 (:time event))))
+
+(defn- format-event
```
Comment (jamtur01) [inline src/riemann/elasticsearch.clj]: Any support for custom fields?

Comment (eguven) [inline src/riemann/elasticsearch.clj]: added in second commit, what do you think? @jamtur01

Comment (jamtur01) [inline src/riemann/elasticsearch.clj]: Sure. Maybe an example of the custom event formatter in the docs?

Comment (eguven) [inline src/riemann/elasticsearch.clj]: done

**Comment by jamtur01** [comment]
Looks good to me - @pyr ?

**Pull request merged**